README.md

```diff
@@ -91,77 +91,192 @@ The front end of the integration is now complete.
+    firstScenarioBuilder.useCaseID = @"YOUR_FIRST_USE_CASE_ID";
+    firstScenarioBuilder.clientSDKID = @"YOUR_FIRST_CLIENT_SDK_ID";
+    firstScenarioBuilder.scenarioID = @"YOUR_FIRST_SCENARIO_ID";
+    firstScenarioBuilder.callbackBackendURL = [NSURL URLWithString:@"YOUR_CALLBACK_URL"];
+    YTBScenario *firstScenario = [firstScenarioBuilder create:&error];
+    [YotiSDK addScenario: firstScenario];
+
+    error = nil;
+    YTBScenarioBuilder *secondScenarioBuilder = [[YTBScenarioBuilder alloc] init];
+    secondScenarioBuilder.useCaseID = @"YOUR_SECOND_USE_CASE_ID";
+    secondScenarioBuilder.clientSDKID = @"YOUR_SECOND_CLIENT_SDK_ID";
+    secondScenarioBuilder.scenarioID = @"YOUR_SECOND_SCENARIO_ID";
+    secondScenarioBuilder.callbackBackendURL = [NSURL URLWithString:@"YOUR_CALLBACK_URL"];
+    YTBScenario *secondScenario = [secondScenarioBuilder create:&error];
+    [YotiSDK addScenario: secondScenario];
+
+    return YES;
+}
+```
+then in your viewController class inside your button IBAction function call the function  : 
+```
+public static func startScenario(for useCaseID: String, with delegate: YotiSDKDelegate) throws``` 
+```
+by passing it the useCaseID and self as delegate like this:
+
+```
+swift:
+@IBAction func yotiButtonDidTouchUpInside(_ sender: YotiButton) {
+	guard let useCaseID = sender.useCaseID else{
+		return
+	}
+	do {
+		UIApplication.shared.isNetworkActivityIndicatorVisible = true
+		try YotiSDK.startScenario(for: useCaseID, with: self)
+    } catch {
+		// Handle error here
+	}
+}
+
+objectiveC: 
+
+- (IBAction)buttonDidTouchUpInside:(YotiButton*)sender {
+	NSString* useCaseID = sender.useCaseID;
+	NSError* error = nil;
+
+	if (![useCaseID isEqual:@""]) {
+		[YotiSDK startScenarioForUseCaseID:useCaseID withDelegate:self error:&error];
+
+		if (error != nil) {
+			NSLog(@"error : %@", error.description);
+		}
+	}
+}
 ```
+
+In Swift your ViewController class should comply to YotiSDKDelegate and to BackendDelegate in order to get the callbacks.
+
+```extension ViewController: YotiSDKDelegate {
+    func yotiSDKDidFail(for useCaseID: String, with error: Error) {
+    	//handle here the error opening of Yoti app
+    }
+
+    func yotiSDKDidSucceed(for useCaseID: String, baseURL: URL?, token: String?, url: URL?) {
+        //Handle here the success of the opening of Yoti app for example by requesting a profile from the backend like below
+        //Get the specific scenario by calling  
+        let scenario = YotiSDK.scenario(for: useCaseID)
+        //request the backend to get the profile linked to a specific scenario by passing the token returned and self as delegate for a call back
+        YotiSDK.callbackBackend(scenario: scenario!, token: token!, with: self)
+    }
+
+    func yotiSDKDidOpenYotiApp() {
+    //Handle specific behaviour if needed when the Yoti App didOpen
+
+    }
+}
+```
+
+In Objective C, your viewController should comply to  YTBSDKDelegate and YTBBackendDelegate like this:
```

Comment:
The documentation stating YTBBackendDelegate is optional is missing

Reply:
done